tests/integration/test_fim/test_files/test_report_changes/test_large_changes.py

```diff
@@ -147,38 +116,35 @@ def test_large_changes(filename, folder, original_size, modified_size, get_confi
                  the test will verify that the generated FIM event contains in its 'content_changes' field the proper
                  value depending on the test case.
 
-    wazuh_min_version: 4.2.0
+    wazuh_min_version: 4.5.0
```

Comment:
The test was skipped for inconsistent, but it should work on 4.2. So, Why did you change the min version?

Reply:
This was changed because it was affected by the DBSync update and messages changed as of 4.5